ui/StatusQ/src/StatusQ/Popups/Dialog/StatusDialog.qml

```diff
@@ -10,6 +10,8 @@ import StatusQ.Core.Theme 0.1
 Dialog {
     id: root
 
+    property string subtitle
```

Comment:
Yeah it seemed like a very much needed API, just because setting the subtitle is very commonplace. Plus you don't have to override the whole `background` (including hooking up the close button) just to display the subtitle. It's orthogonal to the `title` property